release-notes/con-relnotes-techpreview-features.adoc

```diff
@@ -58,10 +61,22 @@ Additional detail on how Red Hat provides support for bundled community dynamic
 
 |Azure Devops|`@backstage/plugin-azure-devops-backend`|Azure DevOps backend plugin that contains the API for retrieving builds and pull requests, that are used by the Azure DevOps frontend plugin.
 
+|Azure Devops Scaffolder|`@backstage/plugin-scaffolder-backend-module-azure`|The azure module for @backstage/plugin-scaffolder-backend.
+
+|Bitbucket Cloud|@backstage/plugin-catalog-backend-module-bitbucket-cloud|A Backstage catalog backend module that helps integrate towards Bitbucket Cloud.
```

Comment:
For this one, the require_valiables is: `BITBUCKET_WORKSPACE`


cf https://github.com/janus-idp/backstage-showcase/blob/1.1.x/dynamic-plugins.default.yaml#L604

Reply:
please use the `suggestion`  feature in GH to make it easier to just push your change into my PR. 

![image](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/227597/9702f6c9-a098-4076-beee-2e5cb7d52db0)

(or ctrl-g)

Reply:
is this what you meant to suggest? https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/75/files#r1525362220 for both the Cloud and Server `plugin-catalog-backend-module-bitbucket-*` entries in the ref guide table?  What about the scaffolders?

Reply:
For cloud yes, for server it is `BITBUCKET_HOST` (see my last comment below)

Reply:
... you're still not using the suggest feature. :D